packages/react-composites/tests/browser/common/utils.ts

```diff
@@ -34,9 +34,7 @@ async function screenshotOnFailure<T>(page: Page, fn: () => Promise<T>): Promise
 export const pageClick = async (page: Page, selector: string): Promise<void> => {
   await page.bringToFront();
   await screenshotOnFailure(page, async () => await page.click(selector, { timeout: PER_STEP_TIMEOUT_MS }));
-
-  // Move the mouse off the screen
-  await page.mouse.move(-1, -1);
+  clickOutsideOfPage(page);
```

Comment:
this won't work for interacting with context menus, (I'm assumming) they will be clicked then instantly dismissed by the clickOutsideOfPage
I don't remember the context of `await page.mouse.move(-1, -1);` (and I'm too lazy to look at the hisotry) but my assumption is that it was added to remove flaky hover styling/behavior -- perhaps that could be worked into your stableScreenshot fn

Reply:
Oh oops. This change is also entirely unrelated to the purpose of this PR.

... and this is the reason e2e tests have been failing, I think.

My saviour!